render/detailed/links.go

```diff
@@ -0,0 +1,169 @@
+package detailed
+
+import (
+	"bytes"
+	"net/url"
+	"strings"
+	"text/template"
+
+	"github.com/weaveworks/scope/probe/docker"
+	"github.com/weaveworks/scope/report"
+
+	"github.com/ugorji/go/codec"
+)
+
+// MetricLink describes a link referencing a metric.
+type MetricLink struct {
+	// References the metric id
+	ID       string `json:"id,omitempty"`
+	Label    string `json:"label"`
+	URL      string `json:"url"`
+	Priority int    `json:"priority"`
+}
+
+// Variable name for the query within the metrics graph url
+const urlQueryVarName = ":query"
+
+var (
+	// As configured by the user
+	metricsGraphURL = ""
```

Comment:
@2opremio I reverted this and created #2740, Bryan mentioned he had a few thoughts on improving it, maybe we can continue the discussion there